benches/two_party_ecdsa/lindell_2017/keygen.rs

```diff
@@ -1,10 +1,7 @@
-#[macro_use]
-extern crate criterion;
-extern crate curv;
-extern crate multi_party_ecdsa;
+use criterion::criterion_main;
```

Comment:
should we do the same in:
https://github.com/kigawas/multi-party-ecdsa/blob/keygen-clippy/benches/two_party_ecdsa/lindell_2017/sign.rs#L2 ?

Reply:
You don't need it in Rust 2018

Reply:
yes, I mean - shouldn't we apply the same change you did *here* to the other file linked

Reply:
I'll fix it